backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/OAuthService.java

```diff
@@ -0,0 +1,55 @@
+package com.canvas.service.helperServices;
+
+import com.canvas.exceptions.CanvasAPIException;
+import com.fasterxml.jackson.databind.JsonNode;
+import okhttp3.Response;
+
+/**
+ * This class handles OAuth requests for generation, validation of access tokens
+ */
+public class OAuthService {
+
+    private final CanvasClientService canvasClientService;
+    //TODO: Add and fetch values from config
```

Comment:
should we implement this TODO before merging to master?

Reply:
I will do it in a separate task, where I will remove such constants and store it in a config file.

Reply:
If this is part of the Oauth process, this should be implemented before merging master. Please fix before we can approve this PR